### PR TITLE
On 1865 replace fat calls with local db

### PR DESCRIPTION
**Pull request by davidgouge**
Changes to remove all calls to the FAT api except those related to gathering standards to update our StandardCollations table.  

Removed the old EpaoImporter function and related Handlers.

Refactored StandardService and removed usage of FAT Api types like Standard and StandardSummary.

### CODE REVIEW

File: src/SFA.DAS.AssessorService.Application.Api/Controllers/StandardServiceController.cs [hunk 1]
```diff
@@ -40,7 +33,7 @@ public StandardServiceController(ILogger<StandardServiceController> logger, ISta
         public async Task<IActionResult> StandardServiceGetAllStandardSummaries()
         {
             _logger.LogInformation($@"Get all StandardSummaries from Standard Service");
```
Comment (martin-hornsby) [inline src/SFA.DAS.AssessorService.Application.Api/Controllers/StandardServiceController.cs]: Do we need this as it used to return a list of StandardSummary
Think the naming of this method going forward will get confusing

File: src/SFA.DAS.AssessorService.Application.Api/Services/StandardService.cs [hunk 2]
```diff
@@ -30,78 +30,33 @@ public StandardService(CacheService cacheService, IAssessmentOrgsApiClient asses
             _standardRepository = standardRepository;
         }
 
-        public async Task<IEnumerable<StandardSummary>> GetAllStandardsV2()
+        public async Task<IEnumerable<StandardCollation>> GetAllStandardsV2()
```
Comment (martin-hornsby) [inline src/SFA.DAS.AssessorService.Application.Api/Services/StandardService.cs]: Do we need it to say V2 still?

File: src/SFA.DAS.AssessorService.Application/Handlers/Search/SearchResultExtensions.cs [hunk 3]
```diff
@@ -17,17 +17,17 @@ static class SearchResultExtensions
     {
         public static List<SearchResult> PopulateStandards(this List<SearchResult> searchResults, IStandardService standardService, ILogger<SearchHandler> logger)
         {
-            var allStandards = standardService.GetAllStandards().Result;
+            var allStandards = standardService.GetAllStandardsV2().Result;
 
             foreach (var searchResult in searchResults)
             {
-                var standard = allStandards.SingleOrDefault(s => s.StandardId == searchResult.StdCode.ToString());
+                var standard = allStandards.SingleOrDefault(s => s.ReferenceNumber == searchResult.StdCode.ToString());
```
Comment (martin-hornsby) [inline src/SFA.DAS.AssessorService.Application/Handlers/Search/SearchResultExtensions.cs]: This looks incorrect. StdCode is usually StandardId

File: src/SFA.DAS.AssessorService.Application/Interfaces/IStandardService.cs [hunk 4]
```diff
@@ -7,12 +7,12 @@ namespace SFA.DAS.AssessorService.Application.Interfaces
 {
     public interface IStandardService
     {
-        Task<IEnumerable<StandardSummary>> GetAllStandardsV2();
-        Task<IEnumerable<Standard>> GetAllStandards();
-        Task<IEnumerable<StandardCollation>> GatherAllStandardDetails();
-        Task<Standard> GetStandard(int standardId);
-        Task<Standard> GetStandard(string referenceNumber);
-        Task<IEnumerable<StandardSummary>> GetAllStandardSummaries();
+        Task<IEnumerable<StandardCollation>> GetAllStandardsV2();
```
Comment (martin-hornsby) [inline src/SFA.DAS.AssessorService.Application/Interfaces/IStandardService.cs]: As mentioned earlier do we really want to call this V2?

File: src/SFA.DAS.AssessorService.Application.Api/Services/StandardService.cs (hunk 2 above)
Comment (davidgouge) [reply src/SFA.DAS.AssessorService.Application.Api/Services/StandardService.cs]: Ahh! Good one, was going to rename that. 👍

File: src/SFA.DAS.AssessorService.Application/Interfaces/IStandardService.cs (hunk 4 above)
Comment (davidgouge) [reply src/SFA.DAS.AssessorService.Application/Interfaces/IStandardService.cs]: ☝️

File: src/SFA.DAS.AssessorService.Application.Api/Controllers/StandardServiceController.cs (hunk 1 above)
Comment (davidgouge) [reply src/SFA.DAS.AssessorService.Application.Api/Controllers/StandardServiceController.cs]: Yes, good point.  Will rename.

File: src/SFA.DAS.AssessorService.Application/Handlers/Search/SearchResultExtensions.cs (hunk 3 above)
Comment (davidgouge) [reply src/SFA.DAS.AssessorService.Application/Handlers/Search/SearchResultExtensions.cs]: Good spot, thanks. 😁

File: src/SFA.DAS.AssessorService.Application.Api/Controllers/StandardServiceController.cs [hunk 5]
```diff
@@ -37,18 +30,18 @@ public StandardServiceController(ILogger<StandardServiceController> logger, ISta
         [SwaggerResponse((int)HttpStatusCode.OK, Type = typeof(IEnumerable<StandardSummary>))]
```
Comment (martin-hornsby) [inline src/SFA.DAS.AssessorService.Application.Api/Controllers/StandardServiceController.cs]: Is this the correct return type?

**Pull request merged**